### PR TITLE
Added support to filter by event type

### PR DESCRIPTION
**Pull request by nathandeamer**

### CODE REVIEW

**Comment by marcells** [comment]
Thank you for your PR.

But could you please add the new setting to the `Readme.md` so that users of the Drone service know that this kind of filter exist? Then I'll merge your PR.

File: app/services/Drone.js [hunk 1]
```diff
@@ -27,10 +27,15 @@ module.exports = function() {
...
     },
     filterBuilds = function(build) {
+      var result = true;
       if (self.configuration.branch) {
-        return build.target === self.configuration.branch;
+        result = build.target === self.configuration.branch;
       }
-      return true;
+
+      if (self.configuration.event) {
```
Comment (marcells) [inline app/services/Drone.js]: Please add a description of this setting to the `Readme.md`.

**Comment by nathandeamer** [comment]
> Thank you for your PR.
> 
> But could you please add the new setting to the `Readme.md` so that users of the Drone service know that this kind of filter exist? Then I'll merge your PR.

Done :)

**Comment by marcells** [comment]
Thank you. 👍

**Pull request merged**